templates/paragraphs/paragraph--wrapped-image.html.twig

```diff
@@ -1,4 +1,6 @@
 {% embed "@molecules/wrapped-image/yds-wrapped-image.twig" with {
+  wrapped_image__style: content.field_style_variation.0['#markup'],
+  wrapped_image__alignment: content.field_style_position.0['#markup'],
```

Comment:
It looks like instead of the `value`, it's sending the `label`

Reply:
Great note, but I think I have this covered in the related PR. In the Drupal upstream, I have changed these field display settings to use the 'key' https://github.com/yalesites-org/yalesites-project/pull/189/files

If you have this branch checked out, then it should have updated your local configuration. So I'm not sure if something is wrong with out build scripts. You can also try a `drush cim` to see if these changes come through. 

This is the fun of working in multiple repos. If you have any feedback to make it easier, we would greatly appreciate it. The current process is okay but not as straightforward as a typical Drupal project.